netmanager/src/App.js

```diff
@@ -74,141 +75,152 @@ if (localStorage.jwtToken) {
   store.dispatch(setOrganization());
 }
 
-class App extends Component {
-  render() {
-    return (
-      <Provider store={store}>
-        <ThemeProvider theme={theme}>
-          <Router>
-            <div className="App">
-              <Route exact path="/" component={Landing} />
-              <Route
+const App = () => {
+
+  useJiraHelpDesk();
+
+  return (
+    <Provider store={store}>
+      <ThemeProvider theme={theme}>
+        <Router>
```

Comment:
Thanks, I guess we might have to move this router code to the Routers folder in the near future. Just to maintain a short and sweet App.js. :)